source/documentation/deploying_services/redis.md

```diff
@@ -8,32 +8,34 @@ Redis is an open source in-memory data store that can be used as a database cach
 
 To set up a Redis service:
 
-1. Run the following in the command line to see what plans are available for Redis:
+1. Run the following in the command line to see what service plans are available for Redis:
 
     ```
     cf marketplace -s redis
     ```
 
-    There are currently two plans available for Redis:
+    Here is an example of the output you will see (the exact service plans will vary):
 
     ```
-    service plan           description                                                                free or paid
-    tiny-clustered-3.2     568MB RAM, clustered (1 shard), single node, no failover, daily backups    paid
-    tiny-unclustered-3.2   568MB RAM, non-clustered, single node, no failover, no backups             paid
+    service plan            description                                                               free or paid
+    tiny-3.2                568MB RAM, single node, no failover, daily backups                        free
+    medium-ha-3.2           6.37GB RAM, highly-available, daily backups                               paid
     ```
 
-    You should use the `tiny-clustered-3.2` plan as it is backed up every day. Refer to the [Redis plans](/deploying_services/redis/#redis-plans) section of the documentation for more information.
+    Refer to the [Redis service plans](/deploying_services/redis/#redis-service-plans) section of the documentation for more information.
```

Comment:
Could have the content from 'Redis service plans' here instead?

Reply:
Prefer to have the Redis service plans information in a separate section as it is supporting information.